### PR TITLE
Migrate all imports of assert_equals_data to use the new location

### PR DESCRIPTION
**Pull request by thijs-obj**
change:
* import for `assert_equals_data` previous: `from tests.functional.bach.test_data_and_utils`, new: `from bach.testing`
  * removed the import from `test.functional.bach`, so there is no way that there are any forgotten imports as those would be broken now.
* small tweak of Makefile

### CODE REVIEW

File: modelhub/Makefile [hunk 1]
```diff
@@ -11,15 +11,15 @@ tests-bigquery:
 # against Postgres
 	mypy modelhub
 	pycodestyle modelhub
-	pytest --all -n 8 --dist loadgroup --big-query tests_modelhub
+	pytest --big-query -n 8 --dist loadgroup tests_modelhub
```
Comment (thijs-obj) [inline modelhub/Makefile:14]: This is a completely unrelated fix. But I had to put it in some PR, so it became this one.

File: bach/tests/functional/bach/test_data_and_utils.py [hunk 2]
```diff
@@ -16,7 +16,6 @@
 from sql_models.constants import DBDialect
 from sql_models.util import is_bigquery, quote_identifier
 from tests.unit.bach.util import get_pandas_df
-from bach.testing import assert_equals_data
```
Comment (KathiaBarahona) [inline bach/tests/functional/bach/test_data_and_utils.py:19]: :+1:
